charts/flink-job/values.yaml

```diff
@@ -29,6 +29,23 @@ imagePullSecrets: []
 # -- Which Flink version to use
 version: v1_16
 
+internalSsl:
+  # -- Whether to use SSL between the job- and taskmanager
+  enabled: true
+  # -- SSL Configuration
+  configuration:
+    # -- The keystore file used for MTLS between the job- and taskmanager
+    keystore: /flinkkeystore/keystore.jks
+    # -- The truststore file used for MTLS between the job- and taskmanager
+    truststore: /flinkkeystore/truststore.jks
+  # -- The volume name used for accessing the key and truststore
+  volumeName: truststore
+  # -- The volumeMount used for accessing the key and truststore
+  volumeMounts:
+    - name: truststore
+      mountPath: /flinkkeystore
+      readOnly: true
```

Comment:
I think this is unnecessary. The volume-mount will not change, and the volume name is not really important, so I think you can hardcode this in the spec